basic-teal/app.R

```diff
@@ -3,14 +3,14 @@ library(teal)
 nest_logo <- "https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/nest.png"
 
 app <- init(
-  data = list(IRIS = iris, MTCARS = mtcars),
+  data = teal_data(IRIS = iris, MTCARS = mtcars, code = c("IRIS <- iris;MTCARS <- mtcars")),
```

Comment:
Agree, Will use `within`. 
Initially, my thought was that this is a nice way to showcase of `code = ""` because we don't showcase it anywhere. But I guess that is why we should showcase `within` so people don't start using `code = ""` everywhere after seeing just this example.